ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/DynamicSSLSocketFactory.java

```diff
@@ -0,0 +1,77 @@
+package org.cru.godtools.articles.aem.service;
+
+import com.annimon.stream.Stream;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocket;
+import javax.net.ssl.SSLSocketFactory;
+
+public class DynamicSSLSocketFactory extends SSLSocketFactory {
```

Comment:
I'll add a Builder into this class so that we can customize it's functionality over time without breaking prior usage. Also, the SSLSocketFactory will need to be moved to gto-support-util